stl/inc/xlocale

```diff
@@ -651,6 +651,20 @@ protected:
     }
 };
 
+template <class _Ty>
+_INLINE_VAR constexpr bool _Is_one_byte_char_like_v = _Is_any_of_v<remove_cv_t<_Ty>, signed char, unsigned char,
+#ifdef __cpp_lib_byte
+    byte,
+#endif // __cpp_lib_byte
+#ifdef __cpp_char8_t
+    char8_t,
+#endif // __cpp_char8_t
+    char>;
+
+template <class _Elem, class _Byte>
+_INLINE_VAR constexpr bool _Is_codecvt_do_always_noconv_v =
+    is_same_v<_Byte, _Elem> || (_Is_one_byte_char_like_v<_Byte> && _Is_one_byte_char_like_v<_Elem>);

```

Comment:
It might be nice to also check for `_Is_two_byte_char_like_v`, for `wchar_t`/`char16_t`?

Reply:
Yes, there is an issue for that: https://github.com/microsoft/STL/issues/605#issuecomment-881972414
but I have not yet explored what needs to be done there to speed up